cime/config/e3sm/config_grids.xml

```diff
@@ -1522,15 +1835,138 @@
       <file grid="lnd" mask="oEC60to30v3wLI">$DIN_LOC_ROOT/share/domains/domain.lnd.ne30np4_oEC60to30v3wLI_mask.170802.nc</file>
       <desc>ne30np4 is Spectral Elem 1-deg grid:</desc>
     </domain>
+    <domain name="ne30np4.pg1">
+      <nx>5400</nx>
+      <ny>1</ny>
+      <file grid="atm|lnd" mask="gx1v6">DUMMY</file>
+      <file grid="ice|ocn" mask="gx1v6">DUMMY</file>
+      <desc>ne30np4.pg1 is Spectral Elem 1-deg grid w/ 1x1 FV physics grid per element:</desc>
+    </domain>
+    <domain name="ne30np4.pg2">
+      <nx>21600</nx>
+      <ny>1</ny>
+      <file grid="atm|lnd" mask="gx1v6">DUMMY</file>
+      <file grid="ice|ocn" mask="gx1v6">DUMMY</file>
+      <desc>ne30np4.pg2 is Spectral Elem 1-deg grid w/ 2x2 FV physics grid per element:</desc>
+    </domain>
+
+    <!--=====================================================================-->
+    <!-- ne32 -->
+    <domain name="ne32np4">
```

Comment:
could you explain why for ne32 grid ne30 domain file is ok?

Reply:
That was just a placeholder. I have a run script that would put the correct file in. This is what I was talking about in my comment above. Do you think it would be better to just take them out? Or maybe just make sure the value is something like "NULL"?

Reply:
Why can't you put the correct file in config_grids.xml?

Reply:
@rljacob, I talk about this in the description above. The problem is that these were relatively short aquaplanet tests that I was spinning up, and I kinda feel like I should do a long spin up if I'm gonna upload the files to support each of these grids. These were mostly just a validation tool that I don't suspect many people will use. The spin up for realistic topography is trickier. Plus, I'd have to generate the topography files, which I suspect we're going tot revisit how this is done, so I don't see a good argument for doing all that work at this point.

Reply:
Ok I see.   It depends what you mean by "supported".  We have lots of resolutions (ne4, ne11) that anyone can build and test with but are not intended for science runs.   Its ok to add more especially if they are useful for testing now or in the future by you or someone else.

Reply:
I initially assumed that either ne40pg2 or ne45pg2 would give the best match to ne30np4. I later found that ne30pg2 did just fine, but it took a lot of iterations of how I approached the convergence test to get to that conclusion, which is part of why I added so many grids.